include/nbl/asset/IDescriptorSetLayout.h

```diff
@@ -214,7 +325,7 @@ class NBL_API IDescriptorSetLayout : public virtual core::IReferenceCounted
 		virtual ~IDescriptorSetLayout() = default;
 
 		core::smart_refctd_dynamic_array<SBinding> m_bindings;
-		core::smart_refctd_dynamic_array<core::smart_refctd_ptr<sampler_type> > m_samplers;
+		core::smart_refctd_dynamic_array<core::smart_refctd_ptr<sampler_type>> m_samplers;
 
 	public:
 		bool isIdenticallyDefined(const IDescriptorSetLayout<sampler_type>* _other) const
```

Comment:
if you added SoA of the following binding metadata to `SBindingRedirect`
```cpp
			core::bitflag<E_CREATE_FLAGS>* createFlags;
			core::bitflag<IShader::E_SHADER_STAGE>* stageFlags;
```
then we could reimplement `isIdenticallDefined` in terms of comparing `SBindingRedirect` and we could get rig of the `==` and `!=` comparison operators from `SBinding`

Reply:
`SBinding::operator<` also removed.
Resolved.